src/components/IconButton/__test__/IconButton.test.js

```diff
@@ -0,0 +1,28 @@
+import { render, screen, fireEvent } from '@testing-library/react';
```

Comment:
This is great. I will skip tests for this review and leave my comments in the PR for the second mini-challenge.